config/master.key

```diff
@@ -0,0 +1 @@
+fbaa33632176ed90164cfe4ff91c38c6
```

Comment:
looks like we have the config that uses this file commented out, should we remove the file?